BATTLES.md

```diff
@@ -0,0 +1,66 @@
+# Battles
+
+A guide to battles.
+
+## Starting Battles
+
+The code below immediately initiates a battle with a level 1 Rattata.
+It can be used before selecting "continue" after loading the FullScreenPokemon index.html file.
+This is useful when repeatedly testing minor edits to battle animations.
```

Comment:
I don't think the 2nd and 3rd sentences should be included. The info feels irrelevant. 

In general, you've included a lot of text on how to test out changes that doesn't actually have anything to do with battles.